Typist/ViewController.swift

```diff
@@ -16,7 +16,7 @@ class ViewController: UIViewController {
         didSet {
             tableView.tableHeaderView = UIView()
             tableView.tableFooterView = UIView()
-            tableView.keyboardDismissMode = .interactive // allow dismissing keyboard interactively
+            tableView.keyboardDismissMode = .interactive // allows dismissing keyboard interactively
```

Comment:
I think this can be assigned on in `didSet` of keyboard's `scrollView`, this keeping boilerplate a bit more lean on developer's part. Thoughts?

Reply:
Agreed